Scripts/Core/EditorVR.Rays.cs

```diff
@@ -211,6 +211,17 @@ void OnProxyActiveChanged(IProxy proxy)
 							m_IntersectionModule.AddTester(tester);
 
 							highlightModule.AddRayOriginForNode(node, rayOrigin);
+
+							var workspaceModule = evr.m_WorkspaceModule;
```

Comment:
How about we turn this into an interface that our Workspace can use. So, instead of getting a single ray origin you get all of them?